src/westpa/analysis/core.py

```diff
@@ -542,9 +542,23 @@ def _get_wtg_parent_ids(self):
 
         return set(wtg_parent_ids)
 
-    @property
-    def wtg_parents(self):
-        """Iterable[Walker or InitialState]: The wtg parents of the walker."""
+    def get_wtg_parents(self, allow_recycled=True):
+        """
+        Iterable[Walker or InitialState]: The parents of the walker according to 
+        the weight transfer graph.
+
+        Parameters
+        ----------
+        allow_recycled : bool
+            A walker that is just created can have weight transfer parents
+            when the parents were merged and recycled in the last iteration.
+            Set to ``False`` to disallow returning weight transfer parents if 
+            the current walker was recycled in the last iteration.
```

Comment:
I'm wondering if this could be made more explicit, e.g.,
```
Parameters
-----------
include_recycling_events : bool, default True
    Whether to include weight transfers from recycled walkers to initial walkers.
```